index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Debas Portfolio | Dbus</title>
+</head>
+<body>
+    <main class="container">
+      <header>
+        <nav class="navbar" id="nav">
+            <a href="#" class="logo">Dbus-G'</a>
+            <div class="menu">
+                <a href="#" class="humberger"><i class="fas fa-bars"></i></a>
+            </div>
+        </nav>
+      </header>
+      <section class="headline-section">
+          <div class="headline-content">
+              <h1>Debas G'ngus</h1>
+              <h2>Software Development</h2>
+              <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me. </p>
+          </div>
+          <img src="./image/horizontal-line.png" alt="horizontal line image">
+          <div class="skill">
+              <ul class="lists"><li>Language</li> 
+                  <li>Javascript</li>
+                  <li>Ruby</li>
+                  <li>Html</li>
+                  <li>Css</li>
+              </ul>
+              <ul class="lists">
+                  <li>Frameworks</li>
+                <li>Bootsrap</li>
```

Comment:
- [ ] In keeping with [HTML and CSS practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) please fix your indentation here